metadata-models/src/main/pegasus/com/linkedin/common/FieldTransformationType.pdl

```diff
@@ -0,0 +1,21 @@
+namespace com.linkedin.common
+
+/**
+ * Type of the transformation involved in generating destination fields from source fields.
+ */
+enum FieldTransformationType{
```

Comment:
there is a space missing between `FieldTransformationType` and `{`. Use intellij to format all files correctly.

Reply:
haha .. PDL formatter doesn't have issues with this space. Nevertheless, including space for readability 👍